OPERATING.md

```diff
@@ -0,0 +1,79 @@
+# Operating a Convene Neighborhood
+
+- [Social Responsibilities](#social-responsibilities)
+  - [As a Cooperative](#as-a-cooperative)
+  - [As a Person](#as-a-person)
+  - [As a Legacy Organization](#as-a-legacy-organization)
+- [Economic Model](#economic-model)
+  - [As a Cooperative](#as-a-cooperative-1)
+  - [As a Person](#as-a-person-1)
+  - [As a Classic Corporation](#as-a-classic-corporation)
+- [Technology Details](#technology-details)
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
```

Comment:
I am a propoent of using the Oxford comma (would go after "building"). While it's not a hill I choose to die on, I want to mention it in case anyone else thinks it would help with reader comprehension. I don't want to comment everywhere it's not present in case avoiding it is an intentional decision. 
(I do want to note that "social, economic, and technological" does use it, but I don't know if consistency really matters in this particular case. 🤷)